### PR TITLE
Created PaymentType.cs Model for #3

### PR DESCRIPTION
**Pull request by Shu52**
# Description
Created PaymentType Model for #3 . This model does not have any dependencies.
 Fixes (#3 Completes model portion of #3, controller portion will still need to completed)
 ## Type of change
 Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
 # Testing Instructions
-In the directory BangazonAPI, use command "dotnet run" to run program. The server should then start and run without errors.
-Make sure all names line up with ERD.(https://drive.google.com/file/d/1Qo9O2oUezNd13Wsr0_VUbUtwwDo4-oAt/view)

 # Checklist:
 - [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Shu52** [comment]
This model is not dependent on other models. Is required for CustomerPayment model.

File: Models/PaymentType.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    //    Structure in Database
+    //    CREATE TABLE PaymentType(
+    //    PaymentTypeId INTEGER NOT NULL PRIMARY KEY,
+    //    PaymentTypeName varchar(15) NOT NULL
+    //);
+    public class PaymentType
+    {
+        [Key]
+        public int PaymentTypeId { get; set; }
+        [StringLength(15, ErrorMessage = "Maxinum lenght is 15 characters"),Required]
```
Comment (stevebrownlee) [inline Models/PaymentType.cs]: Misspelled maximum, and length

File: Models/PaymentType.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace BangazonAPI.Models
+{   
+    //    Structure in Database
```
Comment (stevebrownlee) [inline Models/PaymentType.cs]: This comment block is unnecessary

File: Models/PaymentType.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    //    PaymentTypeName varchar(15) NOT NULL
+    //);
+    public class PaymentType
+    {
+        [Key]
+        public int PaymentTypeId { get; set; }
+        [StringLength(15, ErrorMessage = "Maxinum lenght is 15 characters"),Required]
+        public string PaymentTypeName { get; set; }
+    }
+}
```
Comment (stevebrownlee) [inline Models/PaymentType.cs]: You are missing the CustomerId column

Comment (jordan-castelloe) [reply Models/PaymentType.cs]: @stevebrownlee , they decided as a team to have two separate tables for `PaymentType`and `CustomerPaymentType` . `CustomerPaymentType` will have a customer id and an account number (i.e. "Jordan's Visa"), but `PaymentType` will just have an id and a name. The `CustomerPaymentType` model is coming later- this is just `PaymentType`

File: Models/PaymentType.cs (hunk 1 above)
Comment (Shu52) [reply Models/PaymentType.cs]: Corrected

File: Models/PaymentType.cs (hunk 2 above)
Comment (Shu52) [reply Models/PaymentType.cs]: Removed block

**Comment by Shu52** [comment]
Issues resolved, ok to merge per Jordan

**Pull request merged**